selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsSingleMachineTest.java

```diff
@@ -56,7 +56,7 @@
 /** @author Skoryk Serhii */
 public class WorkspaceDetailsSingleMachineTest {
   private static final String PROJECT_NAME = NameGenerator.generate("project", 4);
-  private static final Map<String, Boolean> EXPECTED_INSTALLERS =
+  private static final ImmutableMap<String, Boolean> EXPECTED_INSTALLERS =
```

Comment:
Could you, please, fix [EXPECTED_VARIABLES](https://github.com/eclipse/che/blob/master/selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsComposeTest.java#L47) constant in the same way?